### PR TITLE
Enhancement/websocket integration

### PR DESCRIPTION
**Pull request by akinsey**
MERGE `ept-websocket-server` as well!

Test that the websocket server starts with the epochtalk server.

### CODE REVIEW

File: modules/ept-frontend/client/services/websocket.js [hunk 1]
```diff
@@ -124,7 +124,6 @@ function(Alert, Auth, NotificationSvc, Session, $window, $rootScope) {
   });
 
   return {
-    init: function() {},
```
Comment (unenglishable) [inline modules/ept-frontend/client/services/websocket.js]: is this function being called anywhere?  just checking.

Comment (akinsey) [reply modules/ept-frontend/client/services/websocket.js]: No

Comment (unenglishable) [reply modules/ept-frontend/client/services/websocket.js]: kk thx

Comment (unenglishable) [reply modules/ept-frontend/client/services/websocket.js]: image

**Pull request merged**